src/Lorentz/Contracts/TZBTC/V0.hs

```diff
@@ -232,8 +233,7 @@ tzbtcDoc = buildLorentzDoc $ do
     , "DX-License-Identifier: LicenseRef-MIT-BitcoinSuisse"
     ]
   contractName "TZBTC" $ do
-    doc $ $mkDGitRevision $ GitRepoSettings $
-      mappend "https://github.com/tz-wrapped/tezos-btc/commit/"
+    doc $ DGitRevisionUnknown
```

Comment:
It should be `contractGeneralDefault`. And the place where we print contract documentation should be updated to do `buildLorentzDocWithGitRev`.

Reply:
@gromakovsky I have made this change, please check.